api/client/client.go

```diff
@@ -1423,6 +1423,8 @@ func (c *Client) GetOIDCConnector(ctx context.Context, name string, withSecrets
 	if err != nil {
 		return nil, trail.FromGRPC(err)
 	}
+	// DELETE IN 11.0.0
+	resp.CheckAndSetRedirectURLs()
```

Comment:
Does this need to be done on the client side? Seems easy to miss. Can the server just always set both for v10?

Reply:
Yeah this can be simplified to just setting `RedirectURL` in `CheckAndSetDefaults`. 

This was originally a roundabout way of making sure `RedirectURL` wouldn't get unmarshalled, but that isn't needed now that its omitted with `[ (gogoproto.jsontag) = "-" ];`

Reply:
The problem is that v9 servers don't set `RedirectURLs`, so when a v10 client calls `GetOIDCConnector`, you'll get:

```
kind: oidc
...
spec:
  ...
  redirect_url: null
```
I can replace this with `CheckAndSetDefaults` and or move this to `tctl`, but the current location should cover the most paths.